scripts/deploySidechain.ts

```diff
@@ -262,18 +290,45 @@ export async function deploySidechainSystem(
         deployOptionsWithCallbacks([poolManagerSetOperator]),
     );
     // Not a constant address
-    const boosterOwner = await deployContractWithCreate2<BoosterOwner, BoosterOwner__factory>(
+    const boosterOwner = await deployContractWithCreate2<BoosterOwnerLite, BoosterOwnerLite__factory>(
         hre,
         create2Factory,
-        new BoosterOwner__factory(deployer),
-        "BoosterOwner",
-        [addresses.daoMultisig, poolManager.address, booster.address, stashFactory.address, ZERO_ADDRESS, true],
+        new BoosterOwnerLite__factory(deployer),
+        "BoosterOwnerLite",
+        [multisigs.daoMultisig, poolManager.address, booster.address, stashFactory.address, ZERO_ADDRESS, true],
         deployOptions,
     );
 
+    const contracts = {
+        voterProxy,
+        booster,
+        boosterOwner,
+        factories: {
+            rewardFactory,
+            stashFactory,
+            tokenFactory,
+            proxyFactory,
+        },
+        poolManager,
+        auraOFT,
+        l2Coordinator,
+    };
+
     let tx: ContractTransaction;
+    // Configure L1 ,L2 Communications
+    if (l1Configuration) {
```

Comment:
If you want to do this here then make it required, the L1 will always be deployed before the L2 so it should always exist.

Reply:
I removed it to simplify the script